contracts/Groups.sol

```diff
@@ -0,0 +1,119 @@
+pragma solidity ^0.4.24;
+
+
+/**
+ * @title Abstract base contract for all groups.
+ */
+contract Group {
+
+    /**
+     * @dev Check if an account belongs to this group.
+     * @param _account the account to check.
+     */
+    function isMember(address _account) public view returns(bool);
+
+    /**
+     * @dev Modifier for restricting access to members of the group.
+     */
+    modifier onlyMember() {
+        require(isMember(msg.sender), "Not a member of this group");
+
+        _;
+    }
+
+    /**
+     * @dev Call a method on a contract in the name of the group.
+     * @param _contract contract address to call.
+     * @param _data ABI encoded function call data.
+     */
+    function forward(address _contract, bytes _data) public payable onlyMember {
+        // solium-disable-next-line security/no-low-level-calls
+        bool result = _contract.call(_data);
+        // solium-disable-next-line security/no-inline-assembly
+        assembly {
+            // Copy the returned data.
+            returndatacopy(0, 0, returndatasize)
+
+            switch result
+            // call returns 0 on error.
+            case 0 { revert(0, returndatasize) }
+            default { return(0, returndatasize) }
+        }
+    }
+}
+
+
+/**
+ * @title Group with fixed members set at creation time.
+ */
+contract FixedGroup is Group {
+    mapping(address => bool) members;
+
+    constructor(address[] _members) public {
```

Comment:
what is the limitation for the number of _members which can be sent to this constructor ?